modules/auxiliary/scanner/gprs/gtp_echo.rb

```diff
@@ -0,0 +1,176 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'bindata'
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::UDPScanner
+
+  def initialize
+    super(
+      'Name'        => 'GTP Echo Scanner',
+      'Description' => %q{
+        This module sends UDP GTP (GTP-U) echo requests to the target RHOSTS and
+        reports on which ones respond, thus identifying General Packet Radio
+        Service (GPRS) servers. This module does not support scanning with SCTP.
+      },
+      'References'  =>
+        [
+          ['URL', 'https://insinuator.net/tag/gtp/'],
+          ['URL', 'https://www.etsi.org/deliver/etsi_ts/129200_129299/129281/08.00.00_60/ts_129281v080000p.pdf']
+        ],
+      'Author'      =>
+        [
+          'Daniel Mende',    # original gtp-scan.py script
+          'Spencer McIntyre' # metasploit module
+        ],
+      'License'     => MSF_LICENSE
+    )
+
+    register_options([
+      OptEnum.new('VERSION', [ true, 'The GTP version to use', '1', ['1', '2'] ]),
```

Comment:
I think it's fine to make these integers?

Reply:
I did try to use all integers for this datastore option (including the comparisons) and it didn't seem to function correctly.

Reply:
You are right. It seems to prefer strings.